### PR TITLE
Feat/home dashboard

### PR DESCRIPTION
**Pull request by Maxycle**
## Changes

| Type | Description |
| - | - |
| Feat | build dashboard

### Comments

Most of the changes are done on the table.

### Steps

1. Clone the project
2. Open 2 tabs in your terminal, cd in nuxt-frontend in one tab, run `yarn install` and `yarn dev`
3. cd in rails-api in the other tab, run `bundle install` and `bundle exec rails s`
4. Go to localhost:8080, click on `Dashboard` in the navbar

### Expectations

- Check that the dashboard matches the design

## Links

**Design:** [Dashboard/Mockup](https://www.figma.com/file/Z0gaYRYxCpaakHoo4i5pMO/ORS-WEBSITE?t=XVhGKPKXnZqpNLrq-0)

**Codebase:** [84](https://unep-wcmc.codebasehq.com/projects/online-reporting-system-redesign/tickets/84)

### CODE REVIEW

**Review by Joseph-Burke — APPROVED** [review]
I've tweaked some formatting, happy to approve.

**However**, I will mark this PR as "Do Not Merge". The best thing to do here is wait until #4 has been merged into `develop`. At that point, the base of this PR will automatically update to `develop`. Otherwise, we run the risk of reviewing the same changes in two different feature branches.

**TL;DR — This PR can be merged once #4 has been merged**
